### PR TITLE
ECS secret stash and toilet

### PR DESCRIPTION
**Pull request by Macoron**
## About the PR 
Updated my old PRs to modern standards.
~~Also fixed minor problem with toilet fixture prototype.~~

**Screenshots**

**Changelog**

### CODE REVIEW

File: Resources/Prototypes/Entities/Structures/Furniture/toilet.yml [hunk 1]
```diff
@@ -20,9 +20,13 @@
...
-    - shape:
-        !type:PhysShapeAabb {}
-      layer: [ Passable ]
+      - shape:
+          !type:PhysShapeAabb
+          bounds: "-0.25,-0.37,0.24,0.49"
+        mask:
+          - Impassable
+          - VaultImpassable
+          - SmallImpassable
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Structures/Furniture/toilet.yml:29]: I already made a PR fixing seats so nuke dis.

**Review by metalgearsloth — APPROVED** [review]

File: Resources/Prototypes/Entities/Structures/Furniture/toilet.yml (hunk 1 above)
Comment (Macoron) [reply Resources/Prototypes/Entities/Structures/Furniture/toilet.yml]: Done

**Comment by Zumorica** [comment]
Conflicts

**Review by metalgearsloth — APPROVED** [review]
Conflicts

**Comment by Macoron** [comment]
Conflicts eliminated, should be good to merge

**Pull request merged**